src/tables.ts

```diff
@@ -3,6 +3,7 @@
 /* eslint-disable key-spacing */
 
 import { ArticulationStruct } from './articulation';
+import { Flow } from './flow';
```

Comment:
It is not required to import `Flow`. This creates also a circular dependency. Please remove it.

@0xfe @ronyeh I just realised that this is a general issue in the code, I will make a PR to resolve the other ones which should use `Tables` rather than `Flow`.

Reply:
As I mentioned in my other comment below, I don't think the circular references can be avoided without some larger restructuring of the code :(

I hope I'm wrong though.

Reply:
Looking a little more, circular dependencies are pervasive throughout the codebase. For example [`types/common.d.ts`](https://github.com/0xfe/vexflow/blob/24f91ee58fa2de3536b56ee90604d2dab36e498c/src/types/common.d.ts#L1) transitively imports `Flow` via [`note.ts`](https://github.com/0xfe/vexflow/blob/24f91ee58fa2de3536b56ee90604d2dab36e498c/src/note.ts#L5).

Reply:
Agree I am working on a PR removing those.